### PR TITLE
Checks for local Username when returning ContactUsername

### LINKED ISSUE #388: Own files, commets, likes data with incorrect username
The `username` property of `ThreadFilesInfo`, `ThreadCommentInfo`, and `ThreadLikesInfo` is coming back as the first part of the peer id for items that are your own. Other peer's data looks correct.

### PR DESCRIPTION
**Pull request by carsonfarmer**
Fixes #388.

### CODE REVIEW

File: core/contacts.go [hunk 1]
```diff
@@ -79,6 +79,10 @@ func (t *Textile) Contacts() ([]ContactInfo, error) {
 // ContactUsername returns the username for the peer id if known
 func (t *Textile) ContactUsername(id string) string {
 	if id == t.node.Identity.Pretty() {
+		username, err := t.Username()
+		if err == nil && *username != "" {
```
Comment (carsonfarmer) [inline core/contacts.go]: `t.Username` returns `*string` here, so is this right?

**Comment by carsonfarmer** [comment]
Here ya go @asutula!

**Comment by carsonfarmer** [comment]
oops!

**Comment by sanderpick** [comment]
oh Carson, you should bump version in core/main.go, package.json, and mobile/package.json

**Comment by asutula** [comment]
So this contact username will find it's way into all the different types we return from the api that include a `username` property?

**Comment by carsonfarmer** [comment]
Yes, this is the only place that (for example) `ThreadFilesInfo`, `ThreadCommentInfo`, and `ThreadLikesInfo` ever get this information...

**Comment by asutula** [comment]
Peeeeerfect. Thanks!

**Pull request merged**